app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurance/InsuranceGraph.kt

```diff
@@ -96,3 +104,17 @@ fun NavGraphBuilder.insuranceGraph(
     }
   }
 }
+
+private val noEnterTransition: AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition = {
+  fadeIn(
+    animationSpec = tween(durationMillis = 300),
+    initialAlpha = 0.99f,
+  )
+}
+
+private val noExitTransition: AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition = {
+  fadeOut(
+    animationSpec = tween(durationMillis = 300),
+    targetAlpha = 0.99f,
+  )
+}
```

Comment:
these were done to avoid a quick flashing that is caused by the default navigation behaviour.

With noExitTransition:

https://github.com/HedvigInsurance/android/assets/90948269/94304589-aae2-4677-9b14-36d9d3b4cf83

With default:

https://github.com/HedvigInsurance/android/assets/90948269/a615e1c7-8603-4d8d-a4fd-fbe66871fd4e

Reply:
Yeah but this is fine. We will fix the card looking like it's in-place with the shared element apis.
Now with removing the animation the rest of the screen also fails to animate, meaning the items at the bottom and this will be different compared to every other screen in the app.
It also changes the animation for if you are not coming from this screen, if that ever happens in some way, instead of only the transition between these two screens.